src/sonic-eventd/rsyslog_plugin/rsyslog_plugin.cpp

```diff
@@ -0,0 +1,79 @@
+#include <ios>
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <regex>
+#include "rsyslog_plugin.h"
+#include "common/logger.h"
+#include "common/json.hpp"
+#include "common/events.h"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+void RsyslogPlugin::onMessage(string msg) {
```

Comment:
Comments like, called on every log message would be helpful

Reply:
Done